tests/Database/DatabaseQueryBuilderTest.php

```diff
@@ -946,7 +946,7 @@ public function testValueMethodReturnsSingleColumn()
     public function testAggregateFunctions()
     {
         $builder = $this->getBuilder();
-        $builder->getConnection()->shouldReceive('select')->once()->with('select count(*) as aggregate from "users"', [], true)->andReturn([['aggregate' => 1]]);
+        $builder->getConnection()->shouldReceive('select')->once()->with('select count(*) as aggregate from (select * from "users") as t', [], true)->andReturn([['aggregate' => 1]]);
```

Comment:
I need to know actual benchmarks of running this query vs running the old query on an actual large database. I can't merge this until those are provided.